homeassistant/components/tis_control/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "tis_control",
+  "name": "TISControl",
+  "codeowners": ["@Mustafa744"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/tis_control",
+  "homekit": {},
+  "iot_class": "local_polling",
+  "requirements": ["TISControlProtocol==0.1.4", "pillow==10.4.0"],
```

Comment:
Why do we require pillow?

Reply:
because this integration should mainly run on a specific board that has LCD to display a logo

Reply:
You mean home assistant has to run on a specific board? Can you link to it?

Reply:
I dont think integrations should be built for specific hardware

Reply:
the integration can run on any raspberry-compatible device 
also i'm checking the host device if it's not a raspberry pi i create a mock display object 
this snippet is from my library 
ps. the integration was tested in the ha core dev container and itw worked flawlessly 

class FakeDisplay:
    def __init__(self):
        pass

    def begin(self):
        pass

    def set_backlight(self, value):
        pass

    def display(self, img):
        pass

Reply:
so basically what i'm trying to say is that yes the integration has some features dedicated to a specific hardware but also if user doesn't have this hardware he can run the integration without any problems.

Also Home Assistant ha a set of integrations detecated to raspberry pi (specific hardware) so .....? 
![image](https://github.com/user-attachments/assets/449e14b2-2d36-4229-bb06-3e61100217e1)